go.mod

```diff
@@ -7,7 +7,7 @@ require (
 	github.com/gin-contrib/pprof v1.4.0
 	github.com/gin-contrib/static v0.0.1
 	github.com/gin-gonic/gin v1.10.0
-	github.com/multiversx/mx-chain-core-go v1.3.0
+	github.com/multiversx/mx-chain-core-go v1.3.1-0.20250402145918-df6b45b4e74f
```

Comment:
update with the new version?

Reply:
will not merge it until final-ish version, I'll update the branch

Reply:
updated